src/sql/workbench/services/connection/browser/connectionWidget.ts

```diff
@@ -877,5 +933,6 @@ export class ConnectionWidget extends lifecycle.Disposable {
 export enum AuthenticationType {
 	SqlLogin = 'SqlLogin',
 	Integrated = 'Integrated',
-	AzureMFA = 'AzureMFA'
+	AzureMFA = 'AzureMFA',
```

Comment:
Whats the differences here? Is there some doc you can reference so I can become familiar with this new auth type.

Reply:
There isn't a doc. The PR and linked issue describe the differences between AzureMFA and AzureMFAAndUser. There are two main differences:

- AzureMFAAndUser specifies both an Azure Account / Tenant _and_ the username.
- AzureMFAAndUser uses the new, optional azureAccount field to store the account drop-down value rather than overriding the username field because it requires both account and username.

In OSS DBs, the database needs both an access token and the username to connect to the database, where the username selects which DB user to connect as. A single AAD identity can have multiple database users associated with it. The main use-case for this is when one DB user is for the individual AAD user and additional users are mapped to 
AAD security groups of which the user is a member.